### ISSUE TITLE
Make internal port number configurable

### ISSUE THREAD

**Issue by ProfX66** [aliparlakci/bulk-downloader-for-reddit#73]
I kept running into the following error when trying to get the Reddit authorization:
```
[WinError 10013] An attempt was made to access a socket in a way forbidden by its access permissions
```

I saw that it was using the port number 1337 and incidentally the Razer Chroma SDK also uses that same port. I was able to get the authorization to work after stopping the server the Razor software starts.

It would be great to have a way for us to configure which port number we want it to use with out needing to fork your project.

**Comment by thorizer** [comment]
same

**Comment by aliparlakci** [comment]
No worries, a quick fix is on its way.

**Comment by aliparlakci** [comment]
Issue resolved on the 1.8 version
